### PR TITLE
various cleanups

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: roles/common.nix [hunk 1]
```diff
@@ -30,9 +30,6 @@
   # Just disable it since we are using telegraf to monitor raid health.
   systemd.services.mdmonitor.enable = false;
 
-  # enable "sar" system activity collection
-  services.sysstat.enable = true;
```
Comment (zowoq) [inline roles/common.nix:34]: Not sure if anyone is looking at the output of this, doesn't look like telegraf collects it?

File: roles/zfs.nix [hunk 2]
```diff
@@ -5,8 +5,6 @@
     autoSnapshot.monthly = 1;
     autoScrub.enable = true;
   };
-  # zfs is not available for latest lts kenrel
-  boot.zfs.enableUnstable = true;
```
Comment (zowoq) [inline roles/zfs.nix:9]: I think this is fine to remove?

File: roles/common.nix (hunk 1 above)
Comment (Mic92) [reply roles/common.nix:34]: I think so.

File: roles/zfs.nix (hunk 2 above)
Comment (Mic92) [reply roles/zfs.nix:9]: yeah.

**Pull request merged**